server/webhook.go

```diff
@@ -217,6 +223,16 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 	})
 }
 
+// validateGitHubWebhookSignature256 returns true if the signature matches the
+// HMAC hex digested SHA256 hash of the body using the given key.
+func validateGitHubWebhookSignature256(signature, key string, body []byte) bool {
```

Comment:
nit: add a unit test for this? Use a real GitHub response if possible.

Reply:
Addressed in https://github.com/bytebase/bytebase/pull/1995/commits/5de5fbe87e0a863f38254b510d9ac86cd559b03a, used a real payload.